generator/integration_tests/tests/golden_kitchen_sink_tracing_stub_test.cc

```diff
@@ -332,8 +332,8 @@ TEST(GoldenKitchenSinkTracingStubTest, AsyncStreamingRead) {
               "google.test.admin.database.v1.GoldenKitchenSink/StreamingRead"),
           SpanWithStatus(opentelemetry::trace::StatusCode::kError, "fail"),
           SpanHasAttributes(
-              OTelAttribute<std::string>("grpc.peer", _),
-              OTelAttribute<std::string>("gl-cpp.status_code", kErrorCode)))));
+              OTelAttribute<std::string>("gl-cpp.status_code", kErrorCode)),
+          Not(SpanHasAttributes(OTelAttribute<std::string>("grpc.peer", _))))));
```

Comment:
nit: I think I would remove the expectation on `grpc.peer`. It is already tested in the tracing stream wrapper classes. I don't think the tracing stub cares about it.

Reply:
done